Sources/Error Handling/BackendErrorCode.swift

```diff
@@ -112,6 +117,11 @@ extension BackendErrorCode {
         case .invalidSubscriberAttributes,
              .invalidSubscriberAttributesBody:
             return .invalidSubscriberAttributesError
+        case .couldNotCreateAlias:
+            return .configurationError
+        case .requestAlreadyInProgress,
+             .subscriberAttributesAreBeingUpdated:
+            return .operationAlreadyInProgressForProductError
```

Comment:
Note that this error matches android's `OperationAlreadyInProgressError` without the `ForProduct` part... I think I prefer Android's naming but I didn't want to cause a breaking change due to this so I just reused the current naming.